### PR TITLE
mac-capture: Fixed display capture with crop to window and manual, for second monitor

### PR DESCRIPTION
**Pull request by derickbailey**
as discussed on the forums here: https://obsproject.com/forum/threads/second-monitor-not-supported-with-display-capture.77174/

I'm brand new to this codebase and building OBS. but my own testing of this shows that the display capture with crop to window and manual is now working on my second (external) monitor with my 2017 MBP. 

let me know what you need from me, to test this and get this verified and built / delivered.

### CODE REVIEW

**Comment by jp9000** [comment]
I notice you're removing the " - dc->frame.origin.y", but not the " - dc->frame.origin.x" on the line above it.  Why?  That seems rather inconsistent.

**Comment by jp9000** [comment]
Let me know when you have a chance to respond to this.  Closing for the mean time because I'm not sure these changes are really accurate as-is and weren't thoroughly investigated/tested with different monitor setups.

**Comment by derickbailey** [comment]
@jp9000 oops - sorry. didn't see the comments on this. my github folder in my email is constantly piling up with stuff.

regarding your question about the `x` not being removed: it isn't an issue on the x-axis. it's only the y-axis that has problems on second monitors. 

removing the `- dc->frame.origin.y` works fine for both primary and second monitors, one or two monitors connected. I've run this modified code a dozen times on my laptop w/ external monitor, and this change appears to be pretty solid to me. 

what additional testing needs to be done? what additional investigation? i'd be happy to help where I can, instead of just having the issue closed. it is a legit issue, and shouldn't be closed like this, IMO.

**Comment by jp9000** [comment]
Have you tested with monitors in all other possible orientations?  For example, it could work when you have your secondary monitor's orientation set as the "right" monitor, but it might not work if you have its orientation up above relative to the primary monitor.  The reason why I'm bothered by this code is the fact that you changed the Y and not the X makes it feel like "this code worked for my specific case, but I don't know about other cases because I didn't/can't test them".

In other words, are you absolutely 100% positive that your code is the correct code and not just the code that works for your specific case?
